pkg/webhook/policy.go

```diff
@@ -307,6 +312,17 @@ func (h *validationHandler) validateConstraint(ctx context.Context, req admissio
 	return false, nil
 }
 
+func (h *validationHandler) validateConfigResource(ctx context.Context, req admission.Request) (bool, error) {
+	gvk := req.AdmissionRequest.Kind
+	correctGK := gvk.Group == "config.gatekeeper.sh" && gvk.Kind == "Config"
```

Comment:
Validating the GVK is redundant, we can avoid checking this twice.

Reply:
Done.  Will leave GVK check to `validateGatekeeperResources()`.